Main.java

```diff
@@ -9,7 +11,33 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    //create a Scanner for user input
+    Scanner input = new Scanner(System.in);
+
+
+    //ask user to submit a number
+    System.out.println("Enter a number to play FizzBuzz");
+
+    //get number from user
+    int number = input.nextInt();
+
+    //figure out what they have to say
+    if(number % 3 == 0 && number % 5 == 0){
+    System.out.println("You should say FizzBuzz");  
+    }else if (number % 3 == 0){
```

Comment:
Watch your indentation. If there is code inside of an if statement, it should be TABBED over an extra bit to show it is inside of the if-statement. Otherwise, good work!